caddyhttp/proxy/upstream_test.go

```diff
@@ -225,4 +227,38 @@ func TestParseBlock(t *testing.T) {
 			}
 		}
 	}
+
+	// tests for insecure skip verify
+	isv_tests := []struct {
+		config string
+		flag   bool
+	}{
+		// Test #1: without flag
+		{"proxy / localhost:8080 {\n health_check / \n}", false},
+
+		// Test #2: with flag
+		{"proxy / localhost:8080 {\n health_check / \n insecure_skip_verify \n}", true},
+	}
+
+	for i, test := range isv_tests {
+		upstreams, err := NewStaticUpstreams(caddyfile.NewDispenser("Testfile", strings.NewReader(test.config)))
+		if err != nil {
+			t.Error("Expected no error. Got:", err.Error())
```

Comment:
As I've suggested below, you can use t.Errorf to create properly-formatted strings with values of arbitrary types.